api/interpret_test.go

```diff
@@ -0,0 +1,57 @@
+package api_test
+
+import (
+	"testing"
+	"os"
+
+	"github.com/nkansal96/aurora-go/api"
+	"github.com/nkansal96/aurora-go/api/backend"
+	"github.com/nkansal96/aurora-go/config"
+	"github.com/nkansal96/aurora-go/errors"
+	"github.com/stretchr/testify/require"
+)
+
+var apiErrorType *errors.APIError
+var c *config.Config
+
+func TestGetInterpretNoCredentials(t *testing.T) {
+	badConfig := &config.Config{ Backend: backend.NewAuroraBackend() }
+
+	_, err := api.GetInterpret(badConfig, "what is the weather in los angeles tomorrow")
+	require.NotNil(t, err)
+	require.IsType(t, apiErrorType, err)
+	require.Subset(t, []string{"MissingApplicationID", "MissingApplicationToken"}, []string{err.(*errors.APIError).Code})
+}
+
+func TestGetInterpretEmptyString(t *testing.T) {
+	_, err := api.GetInterpret(c, "")
+	require.NotNil(t, err)
+	require.IsType(t, apiErrorType, err)
+	require.Subset(t, []string{"APIInvalidInput"}, []string{err.(*errors.APIError).Code})
+}
+
+func TestGetInterpret(t *testing.T) {
```

Comment:
Test it on something simple with a predictable output. Some thing this specific will cause tests to fail if we improve our service. Do `api.GetInterpret(c, "what time is it in los angeles")`. The resulting `intent` should be `"time"` and `entities` should be `{ location: "los angeles" }`